src/Config.UserSecrets/Config.UserSecrets.csproj

```diff
@@ -9,7 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="build\**\*.targets" Pack="true" PackagePath="%(Identity)" />
+    <Content Include="build\netstandard2.0\$(PackageId).props" PackagePath="%(Identity)" />
+    <Content Include="build\netstandard2.0\$(PackageId).targets" PackagePath="%(Identity)" />
```

Comment:
I don't see a targets file. did you add one?

Reply:
It already exists.